reana_cluster/backends/kubernetes/k8s.py

```diff
@@ -69,6 +70,7 @@ def __init__(self,
             current_context from config file will be used.
         :param cephfs: Boolean flag toggling the usage of a cephfs volume as
             storage backend.
+        :param cephfs_size: Int number which represents cephfs volume size (GB)
```

Comment:
What do you think about being more explicit with the variable (and CLI option) name? Something like `cephfs_share_size` & `--cephfs-share-size` | `cephfs_volume_size` `--cephfs-volume-size`.